pkgs/applications/misc/slic3r/default.nix

```diff
@@ -42,10 +42,20 @@ stdenv.mkDerivation rec {
     sed -i 's|"/usr/include/asm-generic/ioctls.h"|<asm-generic/ioctls.h>|g' xs/src/libslic3r/GCodeSender.cpp
   '';
 
-  # note the boost-compile-error is fixed in
-  # https://github.com/slic3r/Slic3r/commit/90f108ae8e7a4315f82e317f2141733418d86a68
-  # this patch can be probably be removed in the next version after 1.3.0
-  patches = lib.optional (lib.versionAtLeast boost.version "1.56.0") ./boost-compile-error.patch;
+  patches = [
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/s/slic3r/1.3.0%2Bdfsg1-5/debian/patches/Drop-error-admesh-works-correctly-on-little-endian-machin.patch";
```

Comment:
Are these permalinks?

Reply:
Nope, though these links seem common in nixpkgs. Would you suggest adding links to archive.org?

Reply:
I do like the idea of old expressions working in the future (for bisecting, archeology or whatnot). What do you think?

Reply:
Updated the links.